packages/cli/src/lib/commands/collect/utils/user-flow/load-flow.unit.test.ts

```diff
@@ -0,0 +1,60 @@
+import { loadFlow } from './load-flow';
+import * as fileHelpers from '../../../../core/file';
+import * as fs from 'node:fs';
+
+jest.mock('node:fs');
+jest.mock('../../../../core/file');
+
+describe('loading user-flow scripts for execution', () => {
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+  })
+
+  it('should throw if ufPath does not exist', () => {
+    const existsSyncSpy = jest.spyOn(fs, 'existsSync').mockReturnValue(false);
+    const loadFlowFn = () => loadFlow({ ufPath: './path' });
+    expect(loadFlowFn).toThrow();
+    expect(existsSyncSpy).toHaveBeenCalled();
+  });
+
+  it('should throw if ufPath points to a file and it does not end in with ts or js', () => {
```

Comment:
It is unclear from this test which file the tested function points to as none is passed to the function/via mocks.